crypto/transactions/builder/base.py

```diff
@@ -38,8 +51,51 @@ def second_sign(self, passphrase):
         self.transaction.signSignature = message.signature
         self.transaction.id = self.transaction.get_id()
 
-    def verify(self):
-        self.transaction.verify()
+    def multi_sign(self, passphrase, index):
+        if not self.transaction.signatures:
+            self.transaction.signatures = []
+        self.set_version(2)
+
+        index = len(self.transaction.signatures) if index == -1 else index
+
+        msg = hashlib.sha256(self.transaction.to_bytes()).digest()
+        secret = unhexlify(PrivateKey.from_passphrase(passphrase).to_hex())
+        signature = hexlify(schnorr.bcrypto410_sign(msg, secret))
+
+        index_formatted = hex(index).replace('x', '')
+        self.transaction.signatures.append(index_formatted + signature.decode())
+
+    def schnorr_verify(self):
+        return self.transaction.verify_schnorr()
+
+    def schnorr_verify_multisig(self):
+        return self.transaction.verify_schnorr_multisig()
+
+    def set_version(self, version=2):
+        self.transaction.version = version
+
+    def set_nonce(self, nonce):
+        self.transaction.nonce = nonce
+
+    def set_amount(self, amount):
+        self.transaction.amount = amount
+
+    def set_network(self, network):
```

Comment:
There is already a `set_network(..)` that sets a global network that it used by the transactions: https://sdk.ark.dev/python/crypto/api-documentation#crypto-configuration-network